openmc/mgxs/mgxs.py

```diff
@@ -2242,28 +2386,35 @@ def tallies(self):
         # Instantiate tallies if they do not exist
         if self._tallies is None:
 
-            # Create the non-domain specific Filters for the Tallies
             group_edges = self.energy_groups.group_edges
             energy = openmc.Filter('energy', group_edges)
             energyout = openmc.Filter('energyout', group_edges)
 
-            # Create a list of scores for each Tally to be created
-            if self.correction == 'P0':
-                scores = ['flux', 'nu-scatter', 'scatter-P1']
-                estimator = 'analog'
-                keys = ['flux', 'scatter', 'scatter-P1']
-                filters = [[energy], [energy, energyout], [energyout]]
-            else:
-                scores = ['flux', 'nu-scatter']
-                estimator = 'analog'
-                keys = ['flux', 'scatter']
-                filters = [[energy], [energy, energyout]]
+            # Create lists of scores, filters for each Tally to be created
+            scores = ['flux']
+            filters = [[energy]]
+            keys = ['flux']
+
+            # Create separate tallies for each moment
```

Comment:
same comment as above re: tallying efficiency is better with `nu-scatter-pN` than with a set of `nu-scatter-n` scores